### ISSUE TITLE
Transport controls bug

### ISSUE THREAD

**Issue by ipeychinov** [kbuffington/Georgia#78]
When I disable the transport controls the theme crashes the next time I start foobar.

```
Error: Spider Monkey Panel v1.4.1 (Georgia: Georgia v2.0.0 by Mordred)
volume_bar is undefined

File: volume.js
Line: 189, Column: 13
Stack trace:
  VolumeBtn/this.on_mouse_move@volume.js:189:13
  on_mouse_move@georgia-main.js:1815:15
```

**Comment by kbuffington** [comment]
Good catch! I just pushed a fix to the repository which will be available in v2.0.2. 

For now you can work around this by re-enabling the transport controls (you'll need to do this in the config file: `transport.enableTransportControls`), then hiding the volume button, _THEN_ hiding the transport controls.

**Comment by kbuffington** [comment]
Actually that's dumb. Just open up the config file and `transport.showVolume` to false. Then reload the theme.
